src/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+namespace BiscuitApp;
```

Comment:
delete this line - no need to declare namespace in file that isn't class/interface/trait

Reply:
resolved